articles/introduction-to-lxd-container-manager/index.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+LXD is a highly effective container manager. It provides an individual experience same as that offered by virtual machines. LXD systems are unique because they operate with Linux containers. This technology was introduced by Canonical Ltd alongside other contributors. LXD is supported by numerous Linux distributions. One of the significant pillars of LXD is a powerful REST API. 
+
+### Goal
+This article discusses the key features of LXD, the benefits, as well as its releases. 
+
+### Understanding what LXD entails
```

Comment:
Before you deep dive into LXD, you could add a few sections explaining about `Containers`, `Virtual machines`/`Hypervisors` or anything that would help the reader understand everything clearly.